Atomic/atomic.py

```diff
@@ -516,6 +518,55 @@ def container_run_command(self):
         command += self.image
         return command
 
+    def ps(self):
+        all_containers = []
+
+        # Collect the system containers
+        for i in self.get_system_containers():
+            container = i["Id"]
+            inspect_stdout = util.check_output(["runc", "state", container])
+            ret = json.loads(inspect_stdout)
+            status = ret["status"]
+            if not self.args.all and status != "running":
+                continue
+
+            image = i['Image']
+            command = ""
+            created = dateparse(ret['created']).strftime("%F %H:%M") # pylint: disable=no-member
+            all_containers.append({"type" : "systemcontainer", "container" : container,
+                                   "image" : image, "command" : command, "created" : created,
+                                   "status" : status})
+
+        # Collect the docker containers
+        for container in [x["Id"] for x in self.d.containers(all=self.args.all)]:
```

Comment:
I cannot use get_containers or get_active_containers as they already include the system containers while I want only the docker containers.  get_(active_)containers must probably be refactored to use the ps command.